tests/dragonfly/cluster_test.py

```diff
@@ -1063,38 +1071,27 @@ async def generate_config():
         keeping = node.slots[num_outgoing:]
         node.next_slots.extend(keeping)
 
-    # some more time fo seeder
-    await asyncio.sleep(1.0)
-
     seeder.stop()
```

Comment:
why dont you stop the seeder after the migration status is finished?

Reply:
I don't see any sense to stop it after finish. To check migration consistency we have counters that are stopped after finish

Reply:
what do you mean we check migration consistency also with seeder using capture

Reply:
if we do seeding after the migration is finished we add keys to the target node. counters are easy to check, but I'm not sure how to check seeder

Reply:
Not sure what do you mean I am not sure how to check the seeder. you do a stop here, you dont know if this stop is after the migration finished.. so it could be that the migration is finished you break after your first step in the while loop bellow and you call capture = await seeder.capture()

So instead of calling the stop here move it bellow the while loop

Reply:
I am confused.. you moved it above the while loop and not bellow. do you want to talk? what is not working?